Gemfile

```diff
@@ -31,6 +31,11 @@ group :code_deploy do
   gem 'aws-sdk', '2.0.13.pre'
 end
 
+group :lambda do
+  gem 'aws-sdk', '2.0.13.pre'
```

Comment:
Is it possible to specify any 2.0.x.pre here? The latest as of this writing is 2.0.18.pre.